ChangeLog

```diff
@@ -2,6 +2,15 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+Bugfix
+   * Restrict usage of error code MBEDTLS_ERR_SSL_WANT_READ to situations
```

Comment:
Not a blocker, but I think it's better to write ChangeLog entries from the perspective of the user first, and developer second if at all. IMO the important information for users is that we fix deadlock situations with event-driven I/O, and it's better if the user doesn't have to read a paragraph about what internal changes we made to fix that before getting to the bit that interests them.